test/behavior/generated-parser-behavior.spec.js

```diff
@@ -2894,10 +2895,10 @@ describe("generated parser behavior", () => {
 
         // The "sum" rule
         expect(parser).to.parse("42*43",                   42 * 43);
-        expect(parser).to.parse("42*43+44*45",             42 * 43 + 44 * 45);
-        expect(parser).to.parse("42*43+44*45+46*47+48*49", 42 * 43 + 44 * 45 + 46 * 47 + 48 * 49);
-        expect(parser).to.parse("42*43-44*45",             42 * 43 - 44 * 45);
-        expect(parser).to.parse("42*43-44*45-46*47-48*49", 42 * 43 - 44 * 45 - 46 * 47 - 48 * 49);
+        expect(parser).to.parse("42*43+44*45",             (42 * 43) + (44 * 45));
```

Comment:
So even basic arithmetic priorities should be highlighted with parens? :)

Reply:
Yep.  I actually agree with this.